releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -4,6 +4,18 @@
 # Microsoft.FeatureManagement.AspNetCore
 [Source code ][source_code_web] | [Package (NuGet)][package_web] | [Samples][samples_web] | [Product documentation][docs]
 
+## 4.0.0-preview2 - March 7, 2024
+
+### Enchancements
+
+* Use flags to enable different service implementations in dependency injection. ([#39](https://github.com/microsoft/FeatureManagement-Dotnet/issues/39)). See more details [here](https://github.com/microsoft/FeatureManagement-Dotnet/tree/preview?tab=readme-ov-file#variants-in-dependency-injection).
+* Additional mechanisms to track targeting within telemetry to improve the connection between published events and metrics. ([#409](https://github.com/microsoft/FeatureManagement-Dotnet/issues/409))
+* Added support for Net8 applications by adding Net8 as a build target. ([#364](https://github.com/microsoft/FeatureManagement-Dotnet/issues/364))
```

Comment:
It's not that we didn't support .NET 8 applications before this change. By adding .NET 8 to the targeting framework, we make sure this library uses .NET 8 libraries/APIs when users build a .NET 8 application.

Reply:
Based it off your previous comment: https://github.com/Azure/AppConfiguration/pull/886#discussion_r1525558270

We added net8 as a build target which means we will run tests against it. We did also update the external libraries from 7.0 to 8.0. The only customer facing change here is: "we're less likely to break a .net8 app now". I'd argue that's "adding support ...", but as you said that makes it sound like it wasn't supported at all before. What about just "Added better support ..."?

Reply:
That works for me- maybe a small tweak to be:

Added support for .NET 8 target framework to the Microsoft.FeatureManagement.AspNetCore package.